compiler/onecc-docker/onecc-docker

```diff
@@ -69,7 +69,7 @@ def main():
         print('Dockerfile successfully built.')
 
     contianer_name = f"onecc_{recent_version.replace('.','_')}"
-    user_cmd = ''.join(sys.argv[1:])
```

Comment:
We have confirmed that the `onecc` subtools does not work in the conventional way. Adds a space.